event/src/unvalidated/event.rs

```diff
@@ -90,6 +90,19 @@ where
         }
     }
 
+    /// Returns the 'id' field of the event, which is the Cid of the stream's init event.
+    /// If this event *is* the init event, then it doesn't know its own Cid and returns None.
```

Comment:
"it doesn't know its own CID" feels wrong xD

Reply:
Haha, open to suggestions for alternate wording!

Reply:
I got nothing, just teasing because it's a silly concept